src/api/handler/handler.js

```diff
@@ -1,17 +1,25 @@
-import {axiosNasaClient} from "../../clients/index.js";
+import {nasaAsteroidService} from "../../services/index.js";
 import {getPreviousPeriod, parseDate} from "./dateUtils.js";
 
+const parseQueryParams = (query) => {
+    return {
+        date: query.date,
+        countOnly: Boolean(query.count),
+        dangerousOnly: Boolean(query["were-dangerous-meteors"]),
+    }
+}
+
 export const handler = async (req, res) => {
-    const dateQuery = req.query.date;
-    const {errorMessage, date} = parseDate(dateQuery);
+    const {date, countOnly, dangerousOnly} = parseQueryParams(req.query);
+    const {errorMessage} = parseDate(date);
```

Comment:
according to it's name, `parseDate` should return parsed date, not error message

Reply:
Yes, you are sure. Thank you!